### ISSUE TITLE
command not found 

### ISSUE THREAD

**Issue by ypriverol** [bigbio/proteomicstmt#8]
@daichengxin the following command is not present: 

.command.sh: line 2: markdown_to_html.py: command not found

see logs

```
https://pipelines.actions.githubusercontent.com/ZStP7jrPXD2Akqywf4jLnNC6il8piwTZTTQFrFFUq9apzTahVW/_apis/pipelines/1/runs/9/signedlogcontent/3?urlExpires=2020-12-14T22%3A09%3A04.8378666Z&urlSigningMethod=HMACV1&urlSignature=OR%2B2QcZJYVzcjkyyJODMZ5i3JcsTSZcRxKWHsrK7fnQ%3D
```

related with PR https://github.com/bigbio/proteomicsTMT/pull/7

**Comment by daichengxin** [comment]
sorry , I will add markdown_to_html.py file and submit together after testing the issue #3

**Comment by ypriverol** [comment]
fixed in this PR #10
